src/bg_mpl_stylesheets/colors.py

```diff
@@ -0,0 +1,28 @@
+from enum import Enum
+
+
+class Colors(str, Enum):
+    BG_BLUE = "#0B3C5D"
+    BG_RED = "#B82601"
+    BG_GREEN = "#1C6B0A"
+    BG_LIGHT_BLUE = "#328CC1"
+    BG_LIGHT_GREY = "#A8B6C1"
+    BG_YELLOW = "#D9B310"
+    BG_BROWN = "#6C5050"
+    BG_BURGUNDY = "#76323F"
+    BG_OLIVE_GREEN = "#626E60"
+    BG_MUTED_OLIVE = "#918770"
+    BG_BEIGE = "#C09F80"
+    BG_GREY = "#B0B0B0FF"
+    # Add more colors as needed
+
+    @classmethod
+    def get_color_name(cls, hex_value: str) -> str:
```

Comment:
This method retrieves the color name given the hex code. 

This method used for generating a plot above.